package.json

```diff
@@ -9,16 +9,16 @@
   },
   "scripts": {
     "start": "node src/index.js",
-    "start:prod": "NODE_ENV=production node src/index.js",
+    "start:prod": "cross-env NODE_ENV=production node src/index.js",
     "start:scheduler": "node src/index.js --schedule",
-    "start:scheduler:prod": "NODE_ENV=production node src/index.js --schedule",
+    "start:scheduler:prod": "cross-env NODE_ENV=production node src/index.js --schedule",
     "refilter": "node src/index.js --refilter-only",
     "setup": "node scripts/setup.js",
     "lint": "./node_modules/.bin/eslint test src services",
     "posttest": "npm run lint",
-    "test": "NODE_ENV=test mocha --file './test/helper.js' --recursive \"./src/**/*.test.js\"",
-    "test:debug": "NODE_ENV=test mocha --file './test/helper.js' --recursive \"./src/**/*.test.js\" --inspect-brk --exit",
-    "validate": "NODE_ENV=validation node --unhandled-rejections=strict ./node_modules/.bin/mocha --delay --file './test/helper.js' scripts/validation/validate.js",
+    "test": "cross-env NODE_ENV=test mocha --file './test/helper.js' --recursive \"./src/**/*.test.js\"",
+    "test:debug": "cross-env NODE_ENV=test mocha --file './test/helper.js' --recursive \"./src/**/*.test.js\" --inspect-brk --exit",
+    "validate": "cross-env NODE_ENV=validation node --unhandled-rejections=strict node_modules/mocha/bin/_mocha --delay --file './test/helper.js' scripts/validation/validate.js",
```

Comment:
Navigating in the module filesystem is not safe (it is akin to using a private API instead of the public one). Are there no binary symlink folders in Windows?